assets/prototype/application/ui/components/input/inlineEditInput/InlineEditInput.tsx

```diff
@@ -4,6 +4,7 @@ interface InlineEditInputProps {
 	component: React.ReactType;
 	defaultValue: string | number;
 	value: string | number;
+	[key: string]: any;
```

Comment:
I saw this in few places, but not very familiar with this approach, please write some more info :)

Reply:
That allows to pass as many attributes you want.

Reply:
But that might cause some troubles since we don't strictly type the incoming props?

Reply:
In my experience with flow we'd have some kind of base type, that would contain shared props, and further other components will extend and add on top of that.

Reply:
Smth like this: https://github.com/ant-design/ant-design/blob/master/components/input/Input.tsx#L15

Reply:
But since we are not using yet antd, I am ok with this loosely typed approach.

Reply:
Sometimes we need to pass extra props, it's useful there.